tests/test_client_enhanced_1.py

```diff
@@ -0,0 +1,494 @@
+#  Copyright (c) 2022 https://reportportal.io .
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License
+import warnings
+from unittest import mock
+import pytest
+
+from reportportal_client.aio.client import ReconnectReason
+from reportportal_client.aio.client import RetryingTask as Task
+from reportportal_client.aio.client import TaskInfo
+from reportportal_client.aio.client import DEFAULT_TASK_TIMEOUT
+from reportportal_client.aio.client import DEFAULT_SHUTDOWN_TASK_TIMEOUT
+from reportportal_client.aio.multiplexer import Multiplexer
+from reportportal_client.aio.tasks import StateChangeTask, KeepAliveTask, PeriodicTask
+from reportportal_client.client import RP, ProjectSettings
+from reportportal_client.core.rp_issues import Issue
+from reportportal_client.core.rp_requests import RPFile
+from tests import REPORT_ENDPOINT_V1, ROOT_URI, PROJECT_NAME, LAUNCH_ID, NO_OP
```

Comment:
not sure where these imports came from, can't find these in the tests/__init__.py